lib/core/utils/assets.dart

```diff
@@ -15,20 +12,20 @@ abstract class Asset {
 }
 
 class ImageAsset extends Asset {
-  const ImageAsset(String file, {String subfolder})
+  const ImageAsset(String file, {required String subfolder})
       : super(
           file: file,
-          path: subfolder == null ? _folder : '$_folder/$subfolder',
```

Comment:
Not everything is under a subfolder. For this reason, we need to protect our old method here.